ui/app/adt/index.html

```diff
@@ -11,8 +12,11 @@
         <link rel="stylesheet" href="../components/ngDialog/css/ngDialog-theme-plain.min.css"/>
 
         <link rel="stylesheet" href="../styles/clinical.css"/>
+        <link rel="stylesheet" href="../micro-frontends-dist/shared.min.css"/>
+        <link rel="stylesheet" href="../micro-frontends-dist/ipd.min.css"/>
+        <link rel="stylesheet" href="../micro-frontends-dist/next-ui.min.css"/>
         <!-- endbuild -->
-        <title>ADT</title>
+        <title>IPD</title>
```

Comment:
@Arjun-Go @binduak I don't believe renaming the page is necessary. If this change was intentional, could you please explain the rationale behind it?